test/specs/shallow-mount.spec.js

```diff
@@ -389,8 +389,7 @@ describeRunIf(process.env.TEST_ENV !== 'node', 'shallowMount', () => {
           AsyncComponent: () => import('~resources/components/component.vue')
         }
       }
-      const wrapper = shallowMount(TestComponent)
-      expect(wrapper.find({ name: 'AsyncComponent' }).exists()).to.equal(true)
```

Comment:
Hi @eddyerburgh , sorry for commenting on such an old PR.

I was wondering why is the assertion for dynamic imports on `shallowMount` were removed? Seems like I cannot simply handles component as dynamic import with `shallowMount` anymore :thinking: